src/main/java/com/liferay/faces/util/logging/LoggerFactory.java

```diff
@@ -33,39 +30,33 @@
 
 	// Private Constants
 	private static final String CLASS_NAME_LOG4J_LOGGER = "org.apache.log4j.Logger";
-
-	// Statically-Initialized Private Constants
-	private static boolean LOG4J_AVAILABLE = false;
+	private static final boolean LOG4J_AVAILABLE;
 
 	static {
 
+		boolean log4jAvailable = true;
+
 		try {
 			Class.forName(CLASS_NAME_LOG4J_LOGGER);
-			LOG4J_AVAILABLE = true;
 
 			try {
-				new LoggerLog4JImpl(CLASS_NAME_LOG4J_LOGGER);
+				LogManager.getLogger(CLASS_NAME_LOG4J_LOGGER);
 			}
 			catch (NoClassDefFoundError e) {
```

Comment:
I don't think this code would ever occur since LoggerLog4JImpl's constructor swallows all exceptions: https://github.com/liferay/liferay-faces-util/blob/master/src/main/java/com/liferay/faces/util/logging/internal/LoggerLog4JImpl.java#L34-L64